examples/Raytracer_Benchmark/RaytraceBenchmark.hpp

```diff
@@ -323,7 +324,8 @@ int runSimulation(const MyMediumProp *volume_container, const vecgeom::cxx::VPla
   COPCORE_CUDA_CHECK(cudaFree(color));
   COPCORE_CUDA_CHECK(cudaFree(sel_vector_d));
   COPCORE_CUDA_CHECK(cudaFree(nselected_hd));
-  COPCORE_CUDA_CHECK(cudaFree(rtdata));
+  rayAlloc.deallocate(rtdata, 1);
+  // COPCORE_CUDA_CHECK(cudaFree(rtdata));
```

Comment:
I think this shouldn't be in this PR either

Reply:
sure, thanks for noticing, these are all leftovers from splitting from #137